module-apps/application-meditation/widgets/TimerProperty.hpp

```diff
@@ -57,6 +59,8 @@ namespace gui
         bool onFocus(bool isFocused) final;
         bool onInput(const InputEvent &inputEvent) final;
         [[nodiscard]] std::chrono::minutes getTime() noexcept;
+        void setTime(std::int32_t newValue);
+        void setOnChangeCallback(std::function<void(std::int32_t newValue)> callback);
```

Comment:
How about creating typedef for `std::function<void(std::int32_t newValue)>`, for instance:
`using onChangeCallback = std::function<void(std::int32_t newValue)>;`

Reply:
done